crates/bevy_ecs/Cargo.toml

```diff
@@ -20,6 +20,7 @@ trace = []
 bevy_tasks = { path = "../bevy_tasks", version = "0.4.0" }
 bevy_utils = { path = "../bevy_utils", version = "0.4.0" }
 bevy_ecs_macros = { path = "macros", version = "0.4.0" }
+fxhash = "0.2"
```

Comment:
Do we need to use `fx_hash`? In `bevy_util` we are already set up for using `ahash`.

Reply:
See reasoning https://github.com/bevyengine/bevy/pull/1119#issuecomment-751361215

Also fxhash used by other dependencies so it needs to be compiled anyway.